.github/workflows/fossa.yml

```diff
@@ -3,7 +3,9 @@ on:
   # Run this workflow every time a new commit pushed to upstream/fork repository.
   # Run workflow on fork repository will help contributors find and resolve issues before sending a PR.
   push:
+    branches: [master]
   pull_request:
+    branches: [master]
```

Comment:
Why it should be restricted to the `master` branch?

Reply:
only trigger ci checks when PR is submitted to karmada-io/karmada master branch.

Reply:
You might didn't see the comments above:
```
# Run this workflow every time a new commit pushed to upstream/fork repository.
  # Run workflow on fork repository will help contributors find and resolve issues before sending a PR.
```

Reply:
done.